docker-compose.yml

```diff
@@ -0,0 +1,29 @@
+version: "3"
+
+services:
+  db:
+    image: "postgres"
+    restart: unless-stopped
+    networks:
+      - "mylocal"
+    env_file:
+      - ".env"
+    volumes:
+      - ./db-data:/var/lib/postgresql/data
+    ports:
+      - 7777:5432
+
+  migrator:
+    networks:
```

Comment:
You can use image `migrate/migrate` and add volume `./migrations:/migrations` without build process

Reply:
Agree, fix it